bug: Few channels are not found

### Bug description

Several channels are not working. Also showing error in windows powershell.

### Steps to reproduce

Step1: Logged in to JioTV Go using browser
Step2: Selected Color TV but no stream available

### Relevant log output

```shell
panic: Response:  {"code":1206,"message":"Sony Playback url not found"}

goroutine 54 [running]:
runtime/debug.Stack()
        runtime/debug/stack.go:26 +0x5e
github.com/gofiber/fiber/v2/middleware/recover.defaultStackTraceHandler(0x4493e5?, {0x9ed580, 0xc0000e0910})
        github.com/gofiber/fiber/v2@v2.52.5/middleware/recover/recover.go:12 +0x25
github.com/jiotv-go/jiotv_go/v3/cmd.JioTVServer.New.func1.1()
        github.com/gofiber/fiber/v2@v2.52.5/middleware/recover/recover.go:31 +0x74
panic({0x9ed580?, 0xc0000e0910?})
        runtime/panic.go:785 +0x132
log.(*Logger).Panicln(0xc00028a210, {0xc000461580?, 0x2?, 0xc0004615a0?})
        log/log.go:317 +0x65
github.com/jiotv-go/jiotv_go/v3/pkg/television.(*Television).Live(0xc000139360, {0xc000352390, 0x3})
        github.com/jiotv-go/jiotv_go/v3/pkg/television/television.go:124 +0x998
github.com/jiotv-go/jiotv_go/v3/internal/handlers.LiveHandler(0xc00036b508)
        github.com/jiotv-go/jiotv_go/v3/internal/handlers/handlers.go:145 +0x78
github.com/gofiber/fiber/v2.(*App).next(0xc00020af08, 0xc00036b508)
        github.com/gofiber/fiber/v2@v2.52.5/router.go:145 +0x1be
github.com/gofiber/fiber/v2.(*Ctx).Next(0xc0002f90b0?)
        github.com/gofiber/fiber/v2@v2.52.5/ctx.go:1034 +0x4d
github.com/gofiber/fiber/v2/middleware/logger.New.func3(0xc00036b508)
        github.com/gofiber/fiber/v2@v2.52.5/middleware/logger/logger.go:119 +0x2ed
github.com/gofiber/fiber/v2.(*Ctx).Next(0xc0003960c0?)
        github.com/gofiber/fiber/v2@v2.52.5/ctx.go:1031 +0x3d
github.com/jiotv-go/jiotv_go/v3/cmd.JioTVServer.CORS.func2(0xc00036b508)
        github.com/jiotv-go/jiotv_go/v3/internal/middleware/cors.go:30 +0x147
github.com/gofiber/fiber/v2.(*Ctx).Next(0xc0003b5b40?)
        github.com/gofiber/fiber/v2@v2.52.5/ctx.go:1031 +0x3d
github.com/jiotv-go/jiotv_go/v3/cmd.JioTVServer.New.func1(0xa158a0?)
        github.com/gofiber/fiber/v2@v2.52.5/middleware/recover/recover.go:43 +0xbb
github.com/gofiber/fiber/v2.(*App).next(0xc00020af08, 0xc00036b508)
        github.com/gofiber/fiber/v2@v2.52.5/router.go:145 +0x1be
github.com/gofiber/fiber/v2.(*App).handler(0xc00020af08, 0x4c2c8f?)
        github.com/gofiber/fiber/v2@v2.52.5/router.go:172 +0x69
github.com/valyala/fasthttp.(*Server).serveConn(0xc000115208, {0xcecb30, 0xc0003bc048})
        github.com/valyala/fasthttp@v1.55.0/server.go:2379 +0xe70
github.com/valyala/fasthttp.(*workerPool).workerFunc(0xc0002530e0, 0xc000440120)
        github.com/valyala/fasthttp@v1.55.0/workerpool.go:224 +0x91
github.com/valyala/fasthttp.(*workerPool).getCh.func1()
        github.com/valyala/fasthttp@v1.55.0/workerpool.go:196 +0x32
created by github.com/valyala/fasthttp.(*workerPool).getCh in goroutine 1
        github.com/valyala/fasthttp@v1.55.0/workerpool.go:195 +0x190

panic: Response:  {"code":1206,"message":"Sony Playback url not found"}

goroutine 54 [running]:
runtime/debug.Stack()
        runtime/debug/stack.go:26 +0x5e
github.com/gofiber/fiber/v2/middleware/recover.defaultStackTraceHandler(0x4493e5?, {0x9ed580, 0xc0000e0970})
        github.com/gofiber/fiber/v2@v2.52.5/middleware/recover/recover.go:12 +0x25
github.com/jiotv-go/jiotv_go/v3/cmd.JioTVServer.New.func1.1()
        github.com/gofiber/fiber/v2@v2.52.5/middleware/recover/recover.go:31 +0x74
panic({0x9ed580?, 0xc0000e0970?})
        runtime/panic.go:785 +0x132
log.(*Logger).Panicln(0xc00028a210, {0xc000461580?, 0x2?, 0xc0004615a0?})
        log/log.go:317 +0x65
github.com/jiotv-go/jiotv_go/v3/pkg/television.(*Television).Live(0xc000139360, {0xc0000ac278, 0x3})
        github.com/jiotv-go/jiotv_go/v3/pkg/television/television.go:124 +0x998
github.com/jiotv-go/jiotv_go/v3/internal/handlers.LiveHandler(0xc00036b508)
        github.com/jiotv-go/jiotv_go/v3/internal/handlers/handlers.go:145 +0x78
github.com/gofiber/fiber/v2.(*App).next(0xc00020af08, 0xc00036b508)
        github.com/gofiber/fiber/v2@v2.52.5/router.go:145 +0x1be
github.com/gofiber/fiber/v2.(*Ctx).Next(0xc0002f90b0?)
        github.com/gofiber/fiber/v2@v2.52.5/ctx.go:1034 +0x4d
github.com/gofiber/fiber/v2/middleware/logger.New.func3(0xc00036b508)
        github.com/gofiber/fiber/v2@v2.52.5/middleware/logger/logger.go:119 +0x2ed
github.com/gofiber/fiber/v2.(*Ctx).Next(0xc0003960c0?)
        github.com/gofiber/fiber/v2@v2.52.5/ctx.go:1031 +0x3d
github.com/jiotv-go/jiotv_go/v3/cmd.JioTVServer.CORS.func2(0xc00036b508)
        github.com/jiotv-go/jiotv_go/v3/internal/middleware/cors.go:30 +0x147
github.com/gofiber/fiber/v2.(*Ctx).Next(0xc000461b10?)
        github.com/gofiber/fiber/v2@v2.52.5/ctx.go:1031 +0x3d
github.com/jiotv-go/jiotv_go/v3/cmd.JioTVServer.New.func1(0xa158a0?)
        github.com/gofiber/fiber/v2@v2.52.5/middleware/recover/recover.go:43 +0xbb
github.com/gofiber/fiber/v2.(*App).next(0xc00020af08, 0xc00036b508)
        github.com/gofiber/fiber/v2@v2.52.5/router.go:145 +0x1be
github.com/gofiber/fiber/v2.(*App).handler(0xc00020af08, 0x4c2c8f?)
        github.com/gofiber/fiber/v2@v2.52.5/router.go:172 +0x69
github.com/valyala/fasthttp.(*Server).serveConn(0xc000115208, {0xcecb30, 0xc0003bc048})
        github.com/valyala/fasthttp@v1.55.0/server.go:2379 +0xe70
github.com/valyala/fasthttp.(*workerPool).workerFunc(0xc0002530e0, 0xc000440120)
        github.com/valyala/fasthttp@v1.55.0/workerpool.go:224 +0x91
github.com/valyala/fasthttp.(*workerPool).getCh.func1()
        github.com/valyala/fasthttp@v1.55.0/workerpool.go:196 +0x32
created by github.com/valyala/fasthttp.(*workerPool).getCh in goroutine 1
        github.com/valyala/fasthttp@v1.55.0/workerpool.go:195 +0x190
```

### Screenshots or videos

_No response_

### Solution

_No response_

### Additional context

_No response_

### Acknowledgements

- [x] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [x] I have written a short but informative title.
- [x] I filled out all of the requested information in this issue properly.

--- a/internal/handlers/drm.go
+++ b/internal/handlers/drm.go
@@ -47,6 +47,16 @@ func getDrmMpd(channelID, quality string) (*DrmMpdOutput, error) {
 		return nil, err
 	}
 
+	// Quick fix for timesplay channels.
+	if liveResult.AlgoName == "timesplay" {
+		return &DrmMpdOutput{
+			PlayUrl:     tv_url,
+			LicenseUrl:  "/drm?auth=" + enc_key + "&channel_id=" + channelID + "&channel=" + channel_enc_url,
+			Tv_url_host: "",
+			Tv_url_path: "",
+		}, nil
+	}
+
 	parsedTvUrl, err := url.Parse(tv_url)
 	if err != nil {
 		utils.Log.Panicln(err)

--- a/pkg/television/television.go
+++ b/pkg/television/television.go
@@ -168,7 +168,7 @@ func (tv *Television) Live(channelID string) (*LiveURLOutput, error) {
 
 	var url string
 	if tv.AccessToken != "" {
-		url = "https://" + JIOTV_API_DOMAIN + "/playback/apis/v1/geturl?langId=6"
+		url = "https://" + JIOTV_API_DOMAIN + "/playback/apis/v1.1/geturl?langId=6"
 		req.Header.Set(headers.AccessToken, tv.AccessToken)
 	} else {
 		req.Header.Set("osVersion", "8.1.0")

--- a/pkg/television/types.go
+++ b/pkg/television/types.go
@@ -84,7 +84,7 @@ type LiveURLOutput struct {
 	VodStitch   bool     `json:"vodStitch"`
 	Mpd         MPD      `json:"mpd"`
 	IsDRM       bool     `json:"isDRM"`
-	ExtId       string   `json:"extId"`
+	ExtID       string   `json:"extId"`
 	AlgoName    string   `json:"algoName"`
 }
 

--- a/pkg/television/types.go
+++ b/pkg/television/types.go
@@ -84,6 +84,8 @@ type LiveURLOutput struct {
 	VodStitch   bool     `json:"vodStitch"`
 	Mpd         MPD      `json:"mpd"`
 	IsDRM       bool     `json:"isDRM"`
+	ExtId       string   `json:"extId"`
+	AlgoName    string   `json:"algoName"`
 }
 
 // CategoryMap represents Categories for channels

--- a/web/views/flow_player_drm.html
+++ b/web/views/flow_player_drm.html
@@ -64,8 +64,8 @@
             retryParameters: {
               maxAttempts: 3,
             },
-            startAtSegmentBoundary: true,
-            alwaysStreamFullSegments: true,
+            bufferingGoal: 5, // 5 seconds
+            lowLatencyMode: true,
           },
         });
 
@@ -81,8 +81,12 @@
             ) {
               const newUrl = new URL(request.uris[0]);
               // add query params host and path
-              newUrl.searchParams.append("host", "{{ .channel_host }}");
-              newUrl.searchParams.append("path", "{{ .channel_path }}");
+              const channelHost = "{{ .channel_host }}";
+              const channelPath = "{{ .channel_path }}";
+              if (channelHost !== "" && channelPath != "") {
+                newUrl.searchParams.append("host", channelHost);
+                newUrl.searchParams.append("path", channelPath);
+              }
               request.uris = [newUrl.toString()];
             }
           });

--- a/web/views/flow_player_drm.html
+++ b/web/views/flow_player_drm.html
@@ -82,7 +82,7 @@
               // add query params host and path
               const channelHost = "{{ .channel_host }}";
               const channelPath = "{{ .channel_path }}";
-              if (channelHost !== "" && channelPath != "") {
+              if (channelHost !== "" && channelPath !== "") {
                 newUrl.searchParams.append("host", channelHost);
                 newUrl.searchParams.append("path", channelPath);
               }

--- a/web/views/flow_player_drm.html
+++ b/web/views/flow_player_drm.html
@@ -65,7 +65,6 @@
               maxAttempts: 3,
             },
             bufferingGoal: 5, // 5 seconds
-            lowLatencyMode: true,
           },
         });
 

